FoursquareTopTests/StubGetVenueDetailsUseCase.swift

```diff
@@ -38,12 +38,9 @@ class StubGetVenueDetailsUseCase : GetVenueDetailsUseCase {
         )
     }
     
-    func givenVenueWithMenu() {
-        self.venue = VenueViewModel.build(menuURL: NSURL(fileURLWithPath: "http://www.karumi.com"))
-    }
-    
-    func givenVenueWithoutMenu() {
-        self.venue = VenueViewModel.build(menuURL: nil)
+    func givenVenue(withMenuURL url : NSURL?)
```

Comment:
In the new api guidelines, you should avoid using the name type in the param.
`func givenVenue(withMenu url : NSURL?)`

At usage the param will have a reference to URL, it will be redundant 
`VenueViewModel.build(menuURL: url)`
become
`VenueViewModel.build(menu: url)`